rqueue-core/src/main/java/com/github/sonus21/rqueue/listener/PostProcessingHandler.java

```diff
@@ -270,18 +274,19 @@ private int getMaxRetryCount(RqueueMessage rqueueMessage, QueueDetail queueDetai
         : rqueueMessage.getRetryCount();
   }
 
-  private void handleFailure(JobImpl job, int failureCount) {
+  private void handleFailure(JobImpl job, int failureCount, Throwable throwable) {
```

Comment:
don't we need to check for getDoNotRetry() classes here too?

Reply:
We don't need here as this  is just to get the stop flag for backoff method

Reply:
yes we need that here too, my bad.